reactor-core/src/main/java/reactor/core/scheduler/LoomBoundedElasticScheduler.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023 VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.core.scheduler;
+
+import java.time.Clock;
+import java.util.Objects;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import reactor.core.Disposable;
+import reactor.core.Scannable;
+
+final class LoomBoundedElasticScheduler
+		implements Scheduler, SchedulerState.DisposeAwaiter<LoomBoundedElasticScheduler.BoundedServices>, Scannable {
+
+	LoomBoundedElasticScheduler(int maxThreads, int maxTaskQueuedPerThread, ThreadFactory threadFactory, long ttlMillis, Clock clock) {
+		throw new UnsupportedOperationException("Unsupported in JDK lower thank 21");
+	}
+
+	/**
+	 * Create a {@link BoundedElasticScheduler} with the given configuration. Note that backing threads
```

Comment:
I suppose we don't need the javadoc here as it's not a public class. Also, it's not accurate, mentioning executors.